### ISSUE TITLE
Convert a problematic accessibility tests to Playwright

### ISSUE THREAD

**Issue by governmentSponsored** [US-Trustee-Program/Bankruptcy-Oversight-Support-Systems#536]
When we encounter a problematic accessibility test (it fails when we know it shouldn't), move it to Playwright, and implement in the pipeline.

Then we can convert the rest if we choose to.

**Comment by jharcey-flexion** [comment]
Correcting close reason from Completed to Not Planned. CAMS-342 was rejected in the Jira backlog. Corrected as part of a repo-wide DORA-metrics audit reconciling GitHub issue state against the authoritative Jira disposition.
